### ISSUE TITLE
IMPROVE: set buttons from flex to inline-felx (Default Material behavior)

### ISSUE THREAD

**Issue by WeslleyDeSouza** [oblique-bit/oblique#142]
### Oblique version

11

### Is this a regression?

Yes

### Screenshots

![image](https://github.com/user-attachments/assets/6301b5e6-41f9-4191-8bef-d71e253491b3)

![image](https://github.com/user-attachments/assets/55d8702c-5864-40d5-b06b-d8da79c5b5d5)

### Description

```true
Please consider changing the display property of .mat-mdc-button-base.ob-button.mat-mdc-icon-button from display: flex; to display: inline-flex;. This adjustment will align the buttons in a single row.
Or remove the display attribute because its delivered by materials. (Its the default value from materials)
```

### Reproduction

```true
add buttons in one row.
```

### Expected behaviour

```true
default angular materials behavior
```

### Proposed solution

```true
display: flex; to display: inline-flex;
```

**Comment by gillerr** [comment]
Hi @WeslleyDeSouza 

You are perfectly correct and I'm amazed that this hasn't been spotted earlier. Thanks for the report, we'll fix it .

**Comment by gillerr** [comment]
Tracked by the ticker OUI-3324 on our internal ticketing system

**Comment by nina-egger** [comment]
Fixed with 12.0.1
